### ISSUE TITLE
Error when generating REST services

### ISSUE THREAD

**Issue by stpaul39** [jGauravGupta/JPAModeler#70]
--------Error when generating source code from JPA diagram.--------------------------------------

Message : org.netbeans.jcode.rest.controller.RESTGenerator.generateEntityController(RESTGenerator.java:256)

StackTrace : 
java.lang.NullPointerException
	at org.netbeans.jcode.rest.controller.RESTGenerator.generateEntityController(RESTGenerator.java:256)
	at org.netbeans.jcode.rest.controller.RESTGenerator.execute(RESTGenerator.java:186)
	at org.netbeans.jcode.generator.internal.util.Util.generateCRUD(Util.java:485)
	at org.netbeans.jcode.generator.internal.ApplicationGenerator.generate(ApplicationGenerator.java:54)
	at org.netbeans.jcode.generator.JEEApplicationGenerator.generate(JEEApplicationGenerator.java:66)
	at org.netbeans.jpa.modeler.source.generator.task.SourceCodeGeneratorTask.exportCode(SourceCodeGeneratorTask.java:106)
	at org.netbeans.jpa.modeler.source.generator.task.SourceCodeGeneratorTask.begin(SourceCodeGeneratorTask.java:78)
	at org.netbeans.jcode.task.AbstractNBTask.beginTask(AbstractNBTask.java:447)
	at org.netbeans.jcode.task.AbstractNBTask.run(AbstractNBTask.java:88)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1443)
	at org.netbeans.modules.openide.util.GlobalLookup.execute(GlobalLookup.java:68)
	at org.openide.util.lookup.Lookups.executeWith(Lookups.java:303)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2058)

--------------------------------------Here is the modeler file--------------------------------------------

ModelerFile : 
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<jpa:entity-mappings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:java="http://jcp.org/en/jsr/detail?id=270" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:jpa="http://java.sun.com/xml/ns/persistence/orm" pkg="com.mc.tsp.tutorbf.entities" v="2.6" thm="PMetroColorScheme" sm="false" xs="false" id="_14762128762967" xsi:schemaLocation="http://java.sun.com/xml/ns/persistence/orm orm_2_1.xsd">
    <jpa:entity xre="false" compositePrimaryKeyClass="IdentifiantPK" abs="false" class="Identifiant" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476212904706">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="login" id="_147621291368010"/>
            <jpa:basic attribute-type="String" visibile="true" name="password" id="_147621292244511"/>
            <jpa:basic attribute-type="String" visibile="true" name="statut" id="_147621293293112"/>
            <jpa:basic attribute-type="java.sql.Date" visibile="true" name="lastconnected" id="_1476298094210120">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="location" id="_1476299045467122"/>
            <jpa:id attribute-type="Long" visibile="true" name="id" id="_14762129047078">
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="Identifiant.findAll">
            <jpa:query>Select e from Identifiant e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Identifiant.findByLogin">
            <jpa:query>Select e from Identifiant e where e.login=:login</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Identifiant.findByPassword">
            <jpa:query>Select e from Identifiant e where e.password=:password</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Identifiant.findByStatut">
            <jpa:query>Select e from Identifiant e where e.statut=:statut</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Identifiant.findByLastconnected">
            <jpa:query>Select i from Identifiant i where i.lastconnected=:lastconnected</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Identifiant.findByLocation">
            <jpa:query>Select i from Identifiant i where i.location=:location</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="UtilisateurPK" abs="false" class="Utilisateur" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476213078285">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="nom" id="_147621309889815">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="prenom" id="_147621311119516">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="email" id="_147621314924318">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="civilite" id="_147621311876517">
                <jpa:bv/>
            </jpa:basic>
            <jpa:one-to-many own="true" collection-type="java.util.List" connected-entity-id="_1476297108046" visibile="true" name="mouchards" id="_1476298621776121"/>
            <jpa:one-to-many own="true" collection-type="java.util.List" connected-entity-id="_1476299184933" visibile="true" name="infoSuspensions" id="_1476299618328127"/>
            <jpa:one-to-one own="true" orp="false" optional="true" primaryKey="false" connected-entity-id="_1476212904706" visibile="true" name="identifiant" id="_147621371296224"/>
            <jpa:one-to-one own="true" primaryKey="false" connected-entity-id="_1476270631032" visibile="true" name="adresse" id="_1476295801049114"/>
            <jpa:id attribute-type="Long" visibile="true" name="id" id="_147621307828613">
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="Utilisateur.findAll">
            <jpa:query>Select e from Utilisateur e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Utilisateur.findByNom">
            <jpa:query>Select u from Utilisateur u where u.nom=:nom</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Utilisateur.findByPrenom">
            <jpa:query>Select u from Utilisateur u where u.prenom=:prenom</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Utilisateur.findByCivilite">
            <jpa:query>Select u from Utilisateur u where u.civilite=:civilite</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Utilisateur.findByEmail">
            <jpa:query>Select u from Utilisateur u where u.email=:email</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="PostulantPK" abs="false" class="Postulant" superclassId="_1476213078285" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476213225278">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="cv" id="_147621324333621">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="photo" id="_147621325627322"/>
            <jpa:one-to-many own="true" collection-type="java.util.List" connected-entity-id="_1476213908692" visibile="true" name="parcourts" id="_147621428282132"/>
            <jpa:one-to-many orp="false" own="true" collection-type="java.util.List" connected-entity-id="_1476299184933" visibile="true" name="infoSuspensionsPost" id="_1476299640327128"/>
            <jpa:one-to-many own="true" collection-type="java.util.List" connected-entity-id="_1476626964264" visibile="true" name="specialisations" id="_1476627489911449"/>
        </jpa:attributes>
        <jpa:nq e="false" name="Postulant.findAll">
            <jpa:query>Select e from Postulant e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Postulant.findByCv">
            <jpa:query>Select p from Postulant p where p.cv=:cv</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Postulant.findByPhoto">
            <jpa:query>Select p from Postulant p where p.photo=:photo</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Postulant.findByAttribute">
            <jpa:query>Select p from Postulant p where p.attribute=:attribute</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="ParcourtPK" abs="false" class="Parcourt" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476213908692">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="diplome" id="_147621394162827">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="etablissement" id="_147621397087328">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="java.sql.Date" visibile="true" name="dateObtension" id="_147621405238829">
                <jpa:bv/>
            </jpa:basic>
            <jpa:id attribute-type="Long" visibile="true" name="id" id="_147621413747930">
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="Parcourt.findAll">
            <jpa:query>Select e from Parcourt e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Parcourt.findByDiplome">
            <jpa:query>Select p from Parcourt p where p.diplome=:diplome</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Parcourt.findByEtablissement">
            <jpa:query>Select p from Parcourt p where p.etablissement=:etablissement</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Parcourt.findByDateObtension">
            <jpa:query>Select p from Parcourt p where p.dateObtension=:dateObtension</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="RecruteurPK" abs="true" class="Recruteur" superclassId="_1476213078285" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476219059581">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="typeRecruteur" id="_147627058134395">
                <jpa:bv/>
            </jpa:basic>
            <jpa:one-to-many orp="false" own="true" collection-type="java.util.List" connected-entity-id="_1476295331570" visibile="true" name="offres" id="_1476295591095113"/>
            <jpa:one-to-many own="true" collection-type="java.util.List" connected-entity-id="_1476299184933" visibile="true" name="infoSuspensions1" id="_147634194433437"/>
        </jpa:attributes>
        <jpa:nq e="false" name="Recruteur.findAll">
            <jpa:query>Select e from Recruteur e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Recruteur.findByTypeRecruteur">
            <jpa:query>Select r from Recruteur r where r.typeRecruteur=:typeRecruteur</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="AdressePK" abs="false" class="Adresse" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476270631032">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="pays" id="_147629502478898">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="ville" id="_147629505157999"/>
            <jpa:basic attribute-type="String" visibile="true" name="secteur" id="_1476295061236100"/>
            <jpa:basic attribute-type="String" visibile="true" name="quartier" id="_1476295076345101"/>
            <jpa:basic attribute-type="String" visibile="true" name="numtel" id="_1476295091360102"/>
            <jpa:basic attribute-type="String" visibile="true" name="longitude" id="_1476295105696103"/>
            <jpa:basic attribute-type="String" visibile="true" name="latitude" id="_1476295120174104">
                <jpa:bv/>
            </jpa:basic>
            <jpa:id attribute-type="Long" visibile="true" name="id" id="_147627063103896">
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="Adresse.findAll">
            <jpa:query>Select e from Adresse e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Adresse.findByPays">
            <jpa:query>Select a from Adresse a where a.pays=:pays</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Adresse.findByVille">
            <jpa:query>Select a from Adresse a where a.ville=:ville</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Adresse.findBySecteur">
            <jpa:query>Select a from Adresse a where a.secteur=:secteur</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Adresse.findByQuartier">
            <jpa:query>Select a from Adresse a where a.quartier=:quartier</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Adresse.findByNumtel">
            <jpa:query>Select a from Adresse a where a.numtel=:numtel</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Adresse.findByLongitude">
            <jpa:query>Select a from Adresse a where a.longitude=:longitude</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Adresse.findByLatitude">
            <jpa:query>Select a from Adresse a where a.latitude=:latitude</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="OffrePK" abs="false" class="Offre" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476295331570">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="titre" id="_1476295387354108"/>
            <jpa:basic attribute-type="String" visibile="true" name="description" id="_1476295473641109"/>
            <jpa:basic attribute-type="java.sql.Date" visibile="true" name="datePoster" id="_1476295484103110">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="statut" id="_1476295539891111"/>
            <jpa:id attribute-type="Long" visibile="true" name="id" id="_1476295331584106">
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="Offre.findAll">
            <jpa:query>Select e from Offre e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Offre.findByTitre">
            <jpa:query>Select e from Offre e where e.titre=:titre</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Offre.findByDescription">
            <jpa:query>Select o from Offre o where o.description=:description</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Offre.findByDatePoster">
            <jpa:query>Select o from Offre o where o.datePoster=:datePoster</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Offre.findByStatut">
            <jpa:query>Select o from Offre o where o.statut=:statut</jpa:query>
        </jpa:nq>
        <jpa:table name="ArchiveOffre"/>
        <jpa:st name="ArchiveTable"/>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="MouchardPK" abs="false" class="Mouchard" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476297108046">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="action" id="_1476297115780117"/>
            <jpa:basic attribute-type="String" visibile="true" name="dateEnregistrer" id="_1476297146998118">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="Motif" id="_1476297186160119"/>
            <jpa:id attribute-type="Long" visibile="true" name="id" id="_1476297108057115">
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="Mouchard.findAll">
            <jpa:query>Select e from Mouchard e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Mouchard.findByAction">
            <jpa:query>Select e from Mouchard e where e.action=:action</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Mouchard.findByDateEnregistrer">
            <jpa:query>Select e from Mouchard e where e.dateEnregistrer=:dateEnregistrer</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Mouchard.findByMotif">
            <jpa:query>Select m from Mouchard m where m.Motif=:Motif</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="InfoSuspensionPK" abs="false" class="InfoSuspension" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476299184933">
        <jpa:ts e="true"/>
        <jpa:hc e="true"/>
        <jpa:eq e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="motif" id="_1476299431284125">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="java.sql.Date" visibile="true" name="dateSuspendu" id="_1476299468620126">
                <jpa:bv/>
            </jpa:basic>
            <jpa:id attribute-type="Long" visibile="true" name="id" id="_1476299184952123">
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="InfoSuspension.findAll">
            <jpa:query>Select e from InfoSuspension e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="InfoSuspension.findByMotif">
            <jpa:query>Select i from InfoSuspension i where i.motif=:motif</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="InfoSuspension.findByDateSuspendu">
            <jpa:query>Select i from InfoSuspension i where i.dateSuspendu=:dateSuspendu</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="AdministrateurPK" abs="false" class="Administrateur" superclassId="_1476213078285" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476618442473">
        <jpa:ts e="true"/>
        <jpa:attributes>
            <jpa:one-to-many orp="false" own="true" collection-type="java.util.List" connected-entity-id="_1476299184933" visibile="true" name="infoSuspensionsAdmin" id="_1476618605245343"/>
        </jpa:attributes>
        <jpa:nq e="false" name="Administrateur.findAll">
            <jpa:query>Select e from Administrateur e</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="AgentTutorPK" abs="false" class="AgentTutor" superclassId="_1476213078285" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476618483517">
        <jpa:ts e="true"/>
        <jpa:attributes>
            <jpa:one-to-many orp="false" own="true" collection-type="java.util.List" connected-entity-id="_1476299184933" visibile="true" name="infoSuspensionsTutor" id="_1476618613870344"/>
        </jpa:attributes>
        <jpa:nq e="false" name="AgentTutor.findAll">
            <jpa:query>Select e from AgentTutor e</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:entity xre="false" compositePrimaryKeyClass="SpecialisationPK" abs="false" class="Specialisation" visibile="true" minimized="false" rootElement="_14762128762967" id="_1476626964264">
        <jpa:ts e="true"/>
        <jpa:attributes>
            <jpa:basic attribute-type="String" visibile="true" name="matiere" id="_1476626990471445">
                <jpa:bv/>
            </jpa:basic>
            <jpa:basic attribute-type="String" visibile="true" name="dateDispoDebut" id="_1476627039287446"/>
            <jpa:basic attribute-type="String" visibile="true" name="deteDispoFin" id="_1476627079789447"/>
            <jpa:basic attribute-type="String" visibile="true" name="classe" id="_1476627148173448"/>
            <jpa:id attribute-type="Long" visibile="true" name="id" id="_1476626964339443">
                <jpa:bv/>
                <jpa:generated-value strategy="AUTO"/>
            </jpa:id>
        </jpa:attributes>
        <jpa:nq e="false" name="Specialisation.findAll">
            <jpa:query>Select e from Specialisation e</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Specialisation.findByMatiere">
            <jpa:query>Select s from Specialisation s where s.matiere=:matiere</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Specialisation.findByDateDispoDebut">
            <jpa:query>Select s from Specialisation s where s.dateDispoDebut=:dateDispoDebut</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Specialisation.findByDeteDispoFin">
            <jpa:query>Select s from Specialisation s where s.deteDispoFin=:deteDispoFin</jpa:query>
        </jpa:nq>
        <jpa:nq e="false" name="Specialisation.findByClasse">
            <jpa:query>Select s from Specialisation s where s.classe=:classe</jpa:query>
        </jpa:nq>
    </jpa:entity>
    <jpa:snp e="true" loc="BEFORE_PACKAGE">/**
 * This file was generated by the JPA Modeler
 */</jpa:snp>
    <jpa:c>
        <ct>java.util.List</ct>
        <ct>java.util.Set</ct>
        <ct>java.util.Map</ct>
        <ct>java.util.Collection</ct>
    </jpa:c>
    <jpa:diagram>
        <plane elementRef="_14762128762967">
            <shape elementRef="_1476212904706">
                <Bounds x="119.0" y="20.0" width="148.0" height="176.0"/>
            </shape>
            <shape elementRef="_1476213078285">
                <Bounds x="443.0" y="20.0" width="151.0" height="252.0"/>
            </shape>
            <shape elementRef="_1476213225278">
                <Bounds x="89.0" y="324.0" width="176.0" height="155.0"/>
            </shape>
            <shape elementRef="_1476213908692">
                <Bounds x="760.0" y="20.0" width="142.0" height="134.0"/>
            </shape>
            <shape elementRef="_1476219059581">
                <Bounds x="804.0" y="345.0" width="158.0" height="113.0"/>
            </shape>
            <shape elementRef="_1476270631032">
                <Bounds x="349.0" y="635.0" width="140.0" height="218.0"/>
            </shape>
            <shape elementRef="_1476295331570">
                <Bounds x="774.0" y="606.0" width="128.0" height="155.0"/>
            </shape>
            <shape elementRef="_1476297108046">
                <Bounds x="89.0" y="709.0" width="149.0" height="134.0"/>
            </shape>
            <shape elementRef="_1476299184933">
                <Bounds x="535.0" y="617.0" width="180.0" height="113.0"/>
            </shape>
            <shape elementRef="_1476618442473">
                <Bounds x="298.0" y="324.0" width="186.0" height="58.0"/>
            </shape>
            <shape elementRef="_1476618483517">
                <Bounds x="557.0" y="324.0" width="179.0" height="58.0"/>
            </shape>
            <shape elementRef="_1476626964264">
                <Bounds x="82.0" y="514.0" width="172.0" height="155.0"/>
            </shape>
        </plane>
    </jpa:diagram>
</jpa:entity-mappings>
```

**Comment by jGauravGupta** [comment]
Please attach the modeler file.

**Comment by stpaul39** [comment]
Hi! I have done an update, the modeler file is now attached.

**Comment by jGauravGupta** [comment]
Reported issue has been fixed , please try new [version 2.6.4](http://jpamodeler.github.io/page.html?l=p/download) .
Note : I have observed some other issue with your ER Diagram design , please fix and share the working updated diagram or let me know , I will fix issue with this current diagram
